Tests/FluentAssertions.Specs/Types/TypeExtensionsSpecs.cs

```diff
@@ -197,7 +208,7 @@ private record struct MyRecordStruct(int Value);
 
     private record struct MyRecordStructWithCustomPrintMembers(int Value)
     {
-        private bool PrintMembers(System.Text.StringBuilder builder)
+        private bool PrintMembers(StringBuilder builder)
```

Comment:
Kind of liked this explicitness here and for `CompilerGenerated` below as we're trying to match _exactly_ what types are used in a `record struct`

Reply:
You'll have to explicitly tell Rider/Resharper not to remove the unnecessary namespace specification.